Automattic/wp-calypso
# Editor: Desktop app style layout

This pull request seeks to implement a redesigned post editor layout, matching the latest sidebar layout revisions made to the rest of Calypso.

Before|After
---|---
![Before](https://cloud.githubusercontent.com/assets/1779930/12127589/1895b6ee-b3c6-11e5-9de6-f8e207009479.png)|![After](https://cloud.githubusercontent.com/assets/1779930/12127439/36612c90-b3c5-11e5-9ecd-1b956a5d87ff.png)

__Design notes:__

Further revisions to the mobile layout are planned, and are not included in the changes here. The mobile layout should continue to display as it does currently without issues.

__Testing instructions:__

This is a significant set of changes that touches many areas of the editor page, and as such there are potentially a number of edge case behaviors that have not been accounted for. Please thoroughly test all common editor workflows!

Particularly, confirm that the following items do not have layout issues in all supported browsers:
- Notices (post publish, email verification)
- Standard and non-standard viewport sizes (including mobile actions sidebar)
- Header featured image
- Visual / HTML edit mode toggling
- Editor toolbar pinning, advanced toggle
